web/src/features/fba/components/AdvisoryMetadata.tsx

```diff
@@ -34,7 +34,9 @@ const AdvisoryMetadata = ({ runType, setRunType, forDate, issueDate }: AdvisoryM
             renderInput={params => <TextField {...params} label="Forecast or Actual" />}
           />{' '}
           <Typography variant="subtitle2">is for {forDate.toISODate()}</Typography>
-          <Typography variant="subtitle2">issued on {!isNull(issueDate) ? issueDate.toISO() : ''}</Typography>
+          <Typography variant="subtitle2">
+            issued on {!isNull(issueDate) ? issueDate.toISO({ includeOffset: false }) : ''}
```

Comment:
I think all dates in our app display according to the user's computer date/time settings, so no need to show the UTC offset.